src/EnergyPlus/SurfaceGeometry.cc

```diff
@@ -3280,52 +3283,64 @@ namespace SurfaceGeometry {
+        //        \memo used for shading elements such as trees
+        //        \memo these items are fixed in space and would not move with relative geometry
+        //        \format vertices
+        //        \min-fields 12
+        //   A1 , \field Name
+        //        \required-field
+        //        \type alpha
+        //        \reference AllShadingAndHTSurfNames
+        //        \reference AllShadingSurfNames
+        //   A2,  \field Transmittance Schedule Name
+        //        \note Transmittance schedule for the shading device, defaults to zero (always opaque)
+        //        \type object-list
+        //        \object-list ScheduleNames
+        //   N1 , \field Number of Vertices
+        //        \note shown with 6 vertex coordinates -- extensible object
+        //        \autocalculatable
+        //        \default autocalculate
+        //        \minimum 3
+        //        \note Rules for vertices are given in GlobalGeometryRules coordinates --
+        //        \note For this object all surface coordinates are in world coordinates.
+        //   N2,  \field Vertex 1 X-coordinate
+        //        \begin-extensible
+        //        \required-field
+        //        \units m
+        //        \type real
+        //   N3-N361; as indicated by the N2 value
+        //
+        // Shading:Building:Detailed,
+        //   \extensible:3 -- duplicate last set of x,y,z coordinates, renumbering please (and changing z terminator to , for all but last)
+        //        \memo used for shading elements such as trees, other buildings, parts of this building not being modeled
+        //        \memo these items are relative to the current building and would move with relative geometry
+        //        \format vertices
+        //        \min-fields 12
+        //   A1 , \field Name
+        //        \required-field
+        //        \type alpha
+        //        \reference AllShadingAndHTSurfNames
+        //        \reference AllShadingSurfNames
+        //   A2,  \field Transmittance Schedule Name
+        //        \note Transmittance schedule for the shading device, defaults to zero (always opaque)
+        //        \type object-list
+        //        \object-list ScheduleNames
+        //   N1 , \field Number of Vertices
+        //        \note shown with 6 vertex coordinates -- extensible object
+        //        \autocalculatable
+        //        \default autocalculate
+        //        \minimum 3
+        //        \note Rules for vertices are given in GlobalGeometryRules coordinates --
+        //        \note For this object all surface coordinates are relative to the building origin (0,0,0)
+        //        \note and will rotate with the BUILDING north axis.
+        //   N2,  \field Vertex 1 X-coordinate
+        //        \begin-extensible
+        //        \required-field
+        //        \units m
+        //        \type real
+        //   N3-N361; as indicated by the N2 value
 
```

Comment:
Update the comments description, since the IDD items quotes are quite out-dated.